tests/js/enactorsTests.js

```diff
@@ -0,0 +1,39 @@
+/*!
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/gpii/universal/LICENSE.txt
+*/
+
+(function ($, fluid) {
+    "use strict";
+
+    fluid.registerNamespace("gpii.tests");
+
+    /**************************
+     * Language Enactor Tests *
+     **************************/
+
+    fluid.defaults("gpii.tests.enactor.lang", {
+        gradeNames: ["gpii.firstDiscovery.enactor.lang", "autoInit"],
+        model: {
+            lang: "en"
+        },
+        invokers: {
+            reloadPage: "fluid.identity"
```

Comment:
Should this actually be an assertion to ensure that the invoker gets triggered.
e.g.
```javascript
jqUnit.assert("The reloadPage Invoker should have fired");
```